### PR TITLE
Added ChainAPI deployment troubleshooting

### PR DESCRIPTION
**Pull request by RyanMakiola**
1. Added a troubleshooting section to the deployments
2. Renamed titles

### CODE REVIEW

File: docs/chainapi/user-guides/deployments.md [hunk 1]
```diff
@@ -148,3 +148,21 @@ Managing an existing deployment How adding / removing / changing APIs version
...
+    [GCP](https://cloud.google.com/)) with the provider you selected under the
+    Cloud Provider Settings on the Settings and Networks page
+3.  You have
+    [blockchain provider account(s)](../../airnode/v0.7/concepts/chain-providers.md)
+    for each of the number of providers you selected under the Networks section
+    on the Settings and Networks page
+4.  You have populated the secrets.env file with all the requested variables
+
+If you have met all the above, and you still have problems, please reach out to
+support@chainapi.com with a description of the problem you are facing and any
```
Comment (Siegrift) [inline docs/chainapi/user-guides/deployments.md]: Maybe make it bold? Or even a mailto link?

Comment (wkande) [reply docs/chainapi/user-guides/deployments.md]: I added the mailto link.

**Review by wkande — APPROVED** [review]
I added the mailto link. Will merge now.

**Comment by wkande** [comment]
We need to figure out how to format via prettier when using the GitHub web based VS code editor. I will fix the formatting when I pull this down with other PR for a deployment tonight.

**Pull request merged**